interface/forms/vitals/C_FormVitals.class.php

```diff
@@ -398,12 +398,20 @@ public function default_action_process()
 
         $weight = $_POST["weight"];
         $height = $_POST["height"];
-        if ($weight > 0 && $height > 0) {
-            $_POST["BMI"] = ($weight / $height / $height) * 703;
+        // Validate input for weight and height
+        if (!is_numeric($weight) || $weight <= 0 || $weight > 1000 || !is_numeric($height) || $height <= 0 || $height > 120) {
+            $GLOBALS['error_message'] = "Invalid weight or height. Please check your input.";
```

Comment:
So you added in an error_message, but you're not showing anywhere that you are using it.